src/pool.rs

```diff
@@ -3,7 +3,7 @@ use error::Error::{self, ArgumentError, OperationError};
 use error::Result;
 
 use connstring::Host;
-use stream::{Stream, StreamConnector};
+use stream::{Stream, ConnectMethod};
```

Comment:
These changes are a mix of updating to use the new name of `StreamConnector` (i.e. `ConnectMethod`) and rustfmt changes.